helix-term/src/ui/editor.rs

```diff
@@ -541,6 +541,44 @@ impl EditorView {
         }
     }
 
+    /// Render bufferline at the top
+    pub fn render_bufferline(editor: &Editor, viewport: Rect, surface: &mut Surface) {
+        let pb = PathBuf::from(SCRATCH_BUFFER_NAME); // default filename to use for scratch buffer
+        surface.clear_with(viewport, editor.theme.get("ui.statusline"));
+        let mut len = 0usize;
+        for doc in editor.documents() {
+            let fname = doc
+                .path()
+                .unwrap_or(&pb)
+                .file_name()
+                .unwrap_or_default()
+                .to_str()
+                .unwrap();
+
+            let style = if view!(editor).doc == doc.id() {
+                editor.theme.get("ui.background")
+            } else {
+                editor.theme.get("ui.statusline")
+            };
+
+            surface.set_string(
+                1 + viewport.x + len as u16,
+                viewport.y,
+                format!(" {} {}", fname, if doc.is_modified() { "● " } else { "" }),
+                style,
+            );
+
+            let (text, offset) = if doc.is_modified() {
+                (format!(" {} ● ", fname), fname.len() + 4)
```

Comment:
```suggestion
                (format!(" {}[+] ", fname), fname.len() + 5)
```

Let's reuse the modified indicator from the status-line for consistency